VERSION

```diff
@@ -0,0 +1 @@
+2.5.2
```

Comment:
No, this part of the `release` target updates and commits that file:

https://github.com/basho/riak-erlang-client/blob/features/lrb/gh-238_hex-publish/Makefile#L31-L33

In fact, if you change that file in advance and commit it, the release target will fail because that file won't have changes to commit, which will cause `git` to barf.